data_model/1.2/device_types/OnOffLight.xml

```diff
@@ -0,0 +1,148 @@
+FOR A PARTICULAR PURPOSE, TITLE OR NONINFRINGEMENT. IN NO EVENT WILL THE
+CONNECTIVITY STANDARDS ALLIANCE BE LIABLE FOR ANY LOSS OF PROFITS, LOSS OF
+BUSINESS, LOSS OF USE OF DATA, INTERRUPTION OF BUSINESS, OR FOR ANY OTHER
+DIRECT, INDIRECT, SPECIAL OR EXEMPLARY, INCIDENTAL, PUNITIVE OR CONSEQUENTIAL
+DAMAGES OF ANY KIND, IN CONTRACT OR IN TORT, IN CONNECTION WITH THIS DOCUMENT
+OR THE INFORMATION CONTAINED HEREIN, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH
+LOSS OR DAMAGE.
+
+All company, brand and product names in this document may be trademarks that
+are the sole property of their respective owners.
+
+This notice and disclaimer must be included on all copies of this document.
+
+Connectivity Standards Alliance
+508 Second Street, Suite 206
+Davis, CA 95616, USA
+
+:xrefstyle: basic
+-->
+<deviceType xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="types types.xsd devicetype devicetype.xsd" id="0x0100" name="On/Off Light" revision="2">
+  <revisionHistory>
+    <revision revision="0" summary="Represents device definitions prior to Zigbee 3.0"/>
+    <revision revision="1" summary="Initial Zigbee 3.0 release"/>
+    <revision revision="2" summary="New data model format and notation"/>
+  </revisionHistory>
+  <classification class="simple" scope="endpoint"/>
+  <clusters>
+    <cluster id="0x0003" name="Identify" side="server">
+      <mandatoryConform/>
+      <commands>
+        <command id="0x0040" name="TriggerEffect">
+          <mandatoryConform/>
+        </command>
+      </commands>
+    </cluster>
+    <cluster id="0x0004" name="Groups" side="server">
+      <mandatoryConform/>
+    </cluster>
+    <cluster id="0x0005" name="Scenes" side="server">
+      <otherwiseConform>
+        <provisionalConform/>
+        <mandatoryConform/>
+      </otherwiseConform>
+      <commands>
+        <command id="0x0040" name="EnhancedAddScene">
+          <otherwiseConform>
+            <provisionalConform/>
+            <mandatoryConform/>
+          </otherwiseConform>
+        </command>
+        <command id="0x0041" name="EnhancedViewScene">
+          <otherwiseConform>
+            <provisionalConform/>
+            <mandatoryConform/>
+          </otherwiseConform>
+        </command>
+        <command id="0x0042" name="CopyScene">
+          <otherwiseConform>
+            <provisionalConform/>
+            <mandatoryConform/>
+          </otherwiseConform>
+        </command>
+      </commands>
+    </cluster>
+    <cluster id="0x0006" name="On/Off" side="server">
+      <mandatoryConform/>
+      <features>
+        <feature code="LT" name="LT">
+          <mandatoryConform/>
+        </feature>
+      </features>
+    </cluster>
+    <cluster id="0x0008" name="Level Control" side="server">
+      <optionalConform/>
+      <features>
+        <feature code="OO" name="OO">
+          <mandatoryConform/>
+        </feature>
+        <feature code="LT" name="LT">
+          <mandatoryConform/>
+        </feature>
```

Comment:
I expected the "code" field to have the ID of the feature map bit, otherwise there is a double indirection needed to lookup the feature.

Reply:
If you want to suggest an update to the schema and discuss with Hasty, that's fine, but this is what we have now. It's a fairly direct representation of what's in the spec, which is intended to be human readable. So we'll need to do the lookup somewhere, it's just a question of whether we do it in alchemy or in the tests. The nice part of this schema is that it's readable by a human. Not that humans tend to do that too much. Current comment aside.